examples/codestral-code-interpreter-ts/README.md

```diff
@@ -0,0 +1,44 @@
+# AI Code Execution with Mistral's Codestral
+
+This project tests capabilities of the new Mistral's model on data analysis tasks, using the [Code Interpreter SDK](https://github.com/e2b-dev/code-interpreter) by E2B. Codestral doesn't support using tools for code execution yet, so in this Python example, we added the code interpreting capabilities. We are going to build an AI agent that performs data analysis tasks on provided data, in a form of a csv file.
```

Comment:
"Codestral doesn't support using tools for code execution yet"

should be

"odestral doesn't support using tools yet"

Reply:
"We are going to build an AI agent that performs data analysis tasks on provided data, in a form of a csv file."

1. Not sure if this is an AI agent but I'll leave it up to you
2. I'd simplify it to "We are going to build an AI agent that performs data analysis on a CSV file."